src/web/nginx.conf

```diff
@@ -54,31 +54,33 @@ http {
     ssl_protocols       TLSv1.2 TLSv1.3;
     ssl_prefer_server_ciphers   on;
     ssl_ciphers "EECDH+ECDSA+AESGCM EECDH+aRSA+AESGCM EECDH+ECDSA+SHA384 EECDH+ECDSA+SHA256 EECDH+aRSA+SHA384 EECDH+aRSA+SHA256 EECDH+aRSA+RC4 EECDH EDH+aRSA HIGH !RC4 !aNULL !eNULL !LOW !3DES !MD5 !EXP !PSK !SRP !DSS";
-    # disabling unwanted http methods
+    
+    # Disabling unwanted http methods
     if ($request_method !~ ^(DELETE|GET|POST)$ ){
       return 405;
     }
-    # prevent clickjack attacks
-    add_header X-Frame-Options "SAMEORIGIN";
-    # prevent XSS attacks
-    add_header X-XSS-Protection "1; mode=block";
-    # Configuring HSTS
-    add_header Strict-Transport-Security 'max-age=31536000; includeSubDomains; preload';
+    
+    # Headers
+    add_header X-Frame-Options "SAMEORIGIN";                                                      # prevent clickjack attacks
+    add_header X-XSS-Protection "1; mode=block";                                                  # prevent XSS attacks
```

Comment:
I think this header is only going to work for non-modern browsers. Its not in the scope of this PR, but we should look at other ways to prevents XSS.

Reply:
I think in the future, in regards to security, we might want to switch to [bunkerized nginx](https://github.com/bunkerity/bunkerized-nginx). It would automate all these security practices. It also automates the SSL cert too